clients/privacy-center/components/modals/privacy-request-modal/PrivacyRequestForm.tsx

```diff
@@ -66,10 +66,17 @@ const usePrivacyRequestForm = ({
         },
       ];
 
-      const handleError = () => {
-        setAlert({
-          status: "error",
-          description: "Your request has failed. Please try again.",
+      const handleError = ({
+        title,
+        error,
+      }: {
+        title: string;
+        error?: any;
+      }) => {
+        toast({
+          title,
+          description: error,
```

Comment:
I wonder if it would make sense to do an `error || fallbackErrorMessage` here? so the other places that call this func below don't have to pass in the same "an unhandled error" occurred msg

Reply:
Piggy backing off of my comment from another toast related feedback.

> I think it's okay to leave it out here. The title is very similar to the original fallbackMessage. I'm using the error field here for more specific responses from the backend. 
 The previous version wasn't doing a title and error message at the same time. So the title is acting as a fallback already and the error message is only used if more data is provided from the backend.

Since the title is already acting as a fallback message and there aren't useful messages being return from the backend I left the error field blank.


And the titles themselves are very similiar but not quite the same. One says

> error occurred while processing

while the other says

> error occurred while creating

If this is a confusing pattern to use for the toast messages maybe we can interate on it in the future once the toast helper is merged into fidesui?